cmd/fleetctl/api.go

```diff
@@ -32,6 +35,10 @@ func unauthenticatedClientFromCLI(c *cli.Context) (*service.Client, error) {
 		return nil, errors.Wrap(err, "error creating Fleet API client handler")
 	}
 
+	if cc.RootCA != "" {
+		fleet = setupRootCA(fleet, cc.RootCA)
```

Comment:
Instead of a separate function, let's do this root CA setup in service.NewClient.